projects/material-dayjs-adapter/src/lib/adapter/dayjs-date-formats.ts

```diff
@@ -5,10 +5,10 @@ import { MatDateFormats } from '@angular/material/core';
  */
 export const MAT_DAYJS_DATE_FORMATS: MatDateFormats = {
   parse: {
-    dateInput: 'MM.DD.YYYY',
+    dateInput: 'L',
```

Comment:
Are we sure this works? Just checked it out on a quick test:
https://codepen.io/SwagGrenade/pen/eYmjdqV?editors=1111

![image](https://user-images.githubusercontent.com/32392635/72370009-e7e21780-36c6-11ea-8041-d8a434818970.png)
![image](https://user-images.githubusercontent.com/32392635/72370021-edd7f880-36c6-11ea-8798-2d78bdb254b0.png)

Reply:
To use "L" (which means localized date format with leading zero) you have to include/extend Day.js with the localization plugin. 
Furthermore, to parse a string it with "L" format, you need to include/extend the customParseFormat plugin.